### ISSUE TITLE
CGAL 5.2.2 breaks CGAL::Geomview_stream on Ubuntu 20.04

### ISSUE THREAD

**Issue by acgetchell** [CGAL/cgal#5822]
## Issue Details

CGAL 5.2.2 breaks CGAL::Geomview_stream on Ubuntu 20.04. It continues to work on macOS 11.

## Source Code

My example is based on https://doc.cgal.org/latest/Geomview/Geomview_2gv_terrain_8cpp-example.html

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits): Linux 64-bit
* Compiler: clang-10, gcc-10
* Release or debug mode: Release and Debug
* Specific flags used (if any):
* CGAL version: 5.2.2
* Boost version: 1.7.5
* Other libraries versions if used (Eigen, TBB, etc.): Eigen 3.3.9, tab 2020_U3#4 from vcpkg

Here's a previously working version using CGAL 5.2:

https://travis-ci.com/github/acgetchell/CDT-plusplus/builds/229436969

which compiled 14 days ago using CGAL 5.2:

https://github.com/acgetchell/CDT-plusplus/runs/2845817175?check_suite_focus=true#step:5:109

However, this commit in vcpkg updated the port to 5.2.2:

https://github.com/microsoft/vcpkg/commit/c5ac711fefa6cacbd6bb6335349e36a765babf87

With the result that my unchanged code here:

https://github.com/acgetchell/CDT-plusplus/blob/develop/src/cdt-gv.cpp

No longer compiles on Linux:

https://travis-ci.com/github/acgetchell/CDT-plusplus/builds/231448678

https://github.com/acgetchell/CDT-plusplus/actions/runs/988370362

But does compile on macOS 11 but not macOS 10.15.

**Comment by maxGimeno** [comment]
You can try to add `-DCGAL_USE_GEOMVIEW`  to your compiler flags

**Comment by lrineau** [comment]
That was indeed been removed by PR https://github.com/CGAL/cgal/pull/5299, and that PR was merged our bug-fix releases 5.1.3 and 5.2.1, and later versions. The PR was supposed to fix our CI issues (at the time when travis-ci.com discontinued its free support for open source projects, we switched to using Github Actions instead). The PR has removed the support for Geomview_stream, probably because it was a problem for our CI (and Geomview is deprecated for years).
